### PR TITLE
17759 CustomerRepository getList does not load custom attribute if the name is "company"

### PR DESCRIPTION
**Pull request by nik4152**
### Description (*)
fix method getList customer repository field Company  form billing address 

### Fixed Issues (if relevant)

1. magento/magento2#17759: M2.2.5 : CustomerRepository::getList() does not load custom attribute if the name is "company"

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=19031) <br/>All committers have signed the CLA.

**Comment by magento-engcom-team** [comment]
Hi @nik4152. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me $VERSION instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation

**Comment by VladimirZaets** [comment]
Hi @nik4152  , please, sign CLA, otherwise, we can't process your pull request

**Review by orlangur — APPROVED** [review]

**Review by orlangur — CHANGES_REQUESTED** [review]
Nowadays all changes must be applied to `2.3-develop` first (see https://devdocs.magento.com/guides/v2.3/contributor-guide/contributing.html#rules). Please prepare a new pull request, this one should be `on hold` until the latter is merged.

**Comment by orlangur** [comment]
Closing due to inactivity. Feel free to reach me out anytime later if you wish to continue work on this pull request and it will be reopened.
Thanks for collaboration!

**Pull request closed without merging**
